sdc/rabbit/exceptions.py

```diff
@@ -0,0 +1,24 @@
+class BadMessageError(Exception):
+    """A bad message is broken in some way that will never be accepted by
+    the endpoing and as such should be rejected (it will still be logged
+    and stored so no data is lost).
```

Comment:
It is explicit in the docstrings that whatever is passed to Consumer as the process parameter, should raise these errors correctly:

Function or method to use for processsing message. Will be passed the body of the message as a string decoded using UTF - 8. Should raise sdc.rabbit.DecryptError, sdc.rabbit.BadMessageError or    
sdc.rabbit.RetryableError on failure, depending on the failure mode.